src/services/PlaylistService.js

```diff
@@ -11,12 +16,13 @@ class PlaylistService extends BaseService {
     return this.repository.findOne({ where: { id, userId } });
   }
 
-  // async addSongToPlaylist(id, songId) {
-  // const playlist = await this.repository.findOne({ where: { id } });
-  // const song = await TYPES.SongRepository.findOne({ where: { songId } });
-  // playlist.songs.push(song);
-  // return this.repository.save(playlist);
-  // }
+  async addSongToPlaylist(id, songId) {
+    const playlist = await this.repository.findOne({ where: { id } });
```

Comment:
the row can be shorthanded because findOne accepts not only conditions but primary keys

Reply:
fixed and improved